### ISSUE TITLE
Better way to specify monthly repetitions

### ISSUE THREAD

**Issue by OmeGak** [indico/indico#1592]
``` yaml
# Ticket imported from Trac
```

We were aware of the naive our naive approach and we already have people complaining about it in tickets. We should find a neat way to address the use case of "last monday" of the month.

We should probably go for a weekday selection without calendar.
- 1st, 2nd, 3rd, 4th or last
- monday, tuesday, ..., sunday

**Comment by pferreir** [comment]
See #1422

**Comment by OmeGak** [comment]
`duplicate`?

**Comment by pferreir** [comment]
Not really. The tickets concern different things, even if they are connected.
